packages/react/src/components/_InputWrapper/Icon.module.css

```diff
@@ -1,10 +1,12 @@
 .icon {
+  align-items: center;
   background: var(--icon-background);
-  padding-right: var(--component-input-border_width-focus);
-  padding-left: var(--component-input-border_width-default);
+  border-bottom-left-radius: var(--border-radius);
+  border-top-left-radius: var(--border-radius);
   display: flex;
-  align-items: center;
-  flex: none;
+  justify-content: center;
+  position: absolute;
+  width: var(--icon-width);
 }
```

Comment:
Was there a specific reason we could not just add `height:100%` here instead of `height` prop?